lib/stylesheet/manager/builder.rb

```diff
@@ -222,7 +222,7 @@ def uploads_digest
     sha1s = []
 
     (theme&.upload_fields || []).map do |upload_field|
-      sha1s << upload_field.upload.sha1
+      sha1s << upload_field.upload&.sha1
     end
```

Comment:
Should we `.compact` on the end here since we are adding the `sha1` even if it's `nil`?

Reply:
I intensionally didn't remove the `nil` values since it won't break the code and also it will make the `Digest::SHA1.hexdigest` method's result more unique.